releases/_posts/2020-09-08-spark-release-3-0-1.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Spark Release 3.0.1
+categories: []
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+
+Spark 3.0.1 is a maintenance release containing stability fixes. This release is based on the branch-3.0 maintenance branch of Spark. We strongly recommend all 3.0 users to upgrade to this stable release.
+
+### Notable changes
+  - [[SPARK-26905]](https://issues.apache.org/jira/browse/SPARK-26905): Revisit reserved/non-reserved keywords based on the ANSI SQL standard
+  - [[SPARK-31220]](https://issues.apache.org/jira/browse/SPARK-31220): repartition obeys spark.sql.adaptive.coalescePartitions.initialPartitionNum when spark.sql.adaptive.enabled
```

Comment:
Could we split this `Notable changes` section into sub-secions like `SQL`, `ML`, ...?

Reply:
seems like we don't have sub-sections in other patch releases either.

Reply:
Ah, ok.